lib/config/options/index.ts

```diff
@@ -520,6 +520,15 @@ const options: RenovateOptions[] = [
     subType: 'string',
     default: null,
   },
+  {
+    name: 'dependencyDashboardOSVVulnerabilitySummary',
+    description:
+      'Control if the Dependency Dashboard issue lists CVEs supplied by [osv.dev](https://osv.dev).',
```

Comment:
I don't know if links in the `description:` field will render properly in our docs page. Let's try it and see what happens. 🙈 If the docs build breaks soon we know where to look. 😄